magda-sleuther-linked-data-rating/src/index.ts

```diff
@@ -1,6 +1,6 @@
 import sleuther from "@magda/sleuther-framework/dist/index";
 import linkedDataAspectDef from "./linkedDataAspectDef";
-import datasetQualityAspectDef from "./linkedDataAspectDef";
+import datasetQualityAspectDef from "@magda/sleuther-framework/dist/common-aspect-defs/datasetQualityAspectDef";
 import onRecordFound from "./onRecordFound";
 import commonYargs from "@magda/sleuther-framework/dist/commonYargs";
 
```

Comment:
Line 15 (if should be optional) or 14: probably we need register `source-link-status` aspect as extra interested aspect for web hooker?

Reply:
And If `dataset-format` will be used as more accurate format source, `dataset-format` should be added to `optinalAspects` as well.

Reply:
@t83714 Yeah I agree there could be some good uses for `dataset-format`. I made those changes you suggested, assuming this is what you meant?

Should we make a common-aspect-def for `linkedDataAspectDef` ?

Reply:
@adamzerella Sorry for the late reply --- missed your comment.
I can't find the new commit from this PR. Maybe you haven't pushed the changes yet?
Once you start to listen dataset-format changes, `magda-sleuther-linked-data-rating/src/onRecordFound.ts` line 48, you probably want to have the logic to use the format info from `dataset-format` when it's available.
By looking at the content of the JSON schema file & original requirements, I think `linked-data-rating` sleuther  should only update `linkedDataAspect` (without touching `datasetQualityAspect`)?  We can confirm this with @AlexGilleran .  You don't need to create a new file for `linkedDataAspectDef`. It's here `magda-sleuther-linked-data-rating/src/linkedDataAspectDef.ts`. However, it doesn't hurt to move it to  `common-aspect-defs` --- as long as we update all the reference.